listen.js

```diff
@@ -0,0 +1,7 @@
+const app = require("./app");
```

Comment:
I assume you've made this so you can check your API through Insomnia?
Although nothing is really wrong with this your tests should be checking everything which you would need to see.